torch/csrc/api/include/torch/fft.h

```diff
@@ -5,25 +5,53 @@
 namespace torch {
 namespace fft {
 
-#ifndef DOXYGEN_SHOULD_SKIP_THIS
```

Comment:
I'm not sure what the impact of removing the detail namespace is on doxygen. @glaringlee - do we need this pattern?

Reply:
@mruberry No, you don't need to follow the pattern. The DOXYGEN_SHOULD_SKIP_THIS is to prevent doxygen to generate the documents for its defined scope.  We don't want doxygen to generate documents for functions that defined in detail namespace, so we put there, this Macro is not related to any specific namespace.

Reply:
Oh neat. So removing the indirection in these bindings is OK?

Reply:
Yes, I think so.